images/bay-php/settings.php

```diff
@@ -242,9 +242,16 @@
   fast404_preboot($settings);
 }
 
-// Temp directory.
-if (getenv('TMP')) {
-  $config['system.file']['path']['temporary'] = getenv('TMP');
+if (empty($settings['file_private_path'])) {
+  $settings['file_private_path'] = 'sites/default/files/private';
+}
+
+// Use mounted fileshare for temp directory unless explicity disabled with
+//  BAY_SHARED_TEMP_FILES=false
+if (getenv("BAY_SHARED_TEMP_FILES") !== "false") {
+  $config['system.file']['path']['temporary'] = sprintf("%s/tmp", $settings['file_private_path']);
+} else {
+  $config['system.file']['path']['temporary'] = getenv("TMPDIR") ?: "/tmp";
```

Comment:
Sensible defaults 👏

Reply:
Although the deleted line refers to `TMP` rather than `TMPDIR`?

Reply:
Your comment prompted me to dig into this - https://en.wikipedia.org/wiki/TMPDIR

Looks like TMPDIR is the most widely used in linux/unix systems. I'll update to that effect.